### PR TITLE
build: use SSH disconnect counter-measures

### PR DESCRIPTION
**Pull request by arbrandes**
In order to try and avoid disconnections, enable client-side SSH counter-measures.

### CODE REVIEW

File: .github/workflows/install.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
   install:
     runs-on: ubuntu-latest
     env:
-      SSH: ssh -o StrictHostKeyChecking=no -i ~/.ssh/sandbox.key ubuntu@next-release.openedx.org
+      SSH: ssh -o StrictHostKeyChecking=no -o TCPKeepAlive=yes -o ServerAliveInterval=120 -i ~/.ssh/sandbox.key ubuntu@next-release.openedx.org
```
Comment (DawoudSheraz) [inline .github/workflows/install.yml:29]: curious: why choose serveralive over clientalive? Or why not both?

Comment (arbrandes) [reply .github/workflows/install.yml:29]: I think clientalive is meant for the server-side configuration.  We can add it there, too!

Comment (DawoudSheraz) [reply .github/workflows/install.yml:29]: Oh, it's alright. I missed the client and server part in docs 🤦🏽 . ServerAlive is on client and vice versa.

**Review by DawoudSheraz — APPROVED** [review]

**Comment by arbrandes** [comment]
I tested it today on a manual SSH connection, and I wasn't disconnected once.  Hope it works for the workflow, too!

**Pull request merged**